tools-local/unix/probe-tool.sh

```diff
@@ -2,13 +2,17 @@
 
 usage()
 {
+    echo ""
```

Comment:
As mentioned in earlier comment. Leading and trailing `echo ""` makes the usage more readable (for me). If there is a convention that says not to then, I can remove the unnecessary `echo ""`